### DISCUSSION TITLE
How do I hide the Search Filter?

### DISCUSSION THREAD

**Discussion by yelnyafacee** [rappasoft/laravel-livewire-tables#409] [Q&A]
Hi, is there any way to hide the Search Filter?

**Comment by rappasoft** [comment]
`public $showSearch = false;`

**Reply by yelnyafacee** [reply]
hi, I am getting: `Symfony\Component\ErrorHandler\Error\FatalError
Type of App\Http\Livewire\Backend\UserProductsTable::$showSearch must be bool (as in class Rappasoft\LaravelLivewireTables\DataTableComponent)`

for:

```
/**
 * Class UsersTable.
 */
class UserProductsTable extends DataTableComponent
{

    public $showSearch = false;
    
    ......
    
}
```

is this correct?

**Reply by rappasoft** [reply]
`public bool $showSearch = false;`
